BlueprintUICommonControls/Sources/AttributedLabel+Environment.swift

```diff
@@ -41,4 +50,8 @@ struct ClosureURLHandler: URLHandler {
     func onTap(url: URL) {
         onTap(url)
     }
+
+    func isEquivalent(to other: URLHandler) -> Bool {
```

Comment:
If I change the logic for the `onTap` between updates, will this handler still be re-applied to the backing view (even though it's considered equal)?

Reply:
Yes it will get re-applied. The equality check is only used to cache the `ElementState` however the `viewDescription` will always be called regardless of the element's cache state.